Makefile

```diff
@@ -40,13 +40,11 @@ ifeq (, $(shell which ginkgo 2> /dev/null))
 	$(call go-install-tool,$(GOVER),github.com/sozorogami/gover)
 endif
 
-GINKGO = $(shell pwd)/bin/ginkgo
 ginkgo:
 ifeq (, $(shell which ginkgo 2> /dev/null))
```

Comment:
You're right: if ginkgo is not installed this will fail. Do you know why do we install the binaries in `$(PROJECT_PATH)/bin` instead of `$(GOPATH)/bin`?
I'll revert the change, but it causes problems when I build it because I already have installed the binaries in `$(GOPATH)/bin` and the script can't find them there.

Reply:
I don't know what was the original intention. I had issues with it as well.